Website/DesktopModules/Hotcakes/Hotcakes.Modules.csproj

```diff
@@ -3789,6 +3789,7 @@
     </Content>
     <Content Include="Providers\DataProviders\SqlDataProvider\03.06.00.SqlDataProvider" />
     <Content Include="Providers\DataProviders\SqlDataProvider\03.09.00.SqlDataProvider" />
+    <Content Include="Providers\DataProviders\SqlDataProvider\03.09.01.SqlDataProvider" />
```

Comment:
@ArielBlanco1990 This update is going to be for the 3.9.0 release, not 3.9.1.

Reply:
@WillStrohl This is now resolved.